src/config.ts

```diff
@@ -32,7 +34,7 @@ export interface SnowpackConfig {
     nomodule?: string;
     nomoduleOutput: string;
     optimize: boolean;
-    nodeEnv?: string;
+    env?: EnvVarReplacements | string[];
```

Comment:
that `string[]` is needed because of the way `CLIFlags` extends `SnowpackConfig.installOptions` and CLI `--env` args end up collected as `string[]`. I've tried removing it, but would have to refactor types to allow `env` in CLIFlags to be string[]. E.g.

```ts
export interface CLIFlags
  extends Exclude<Partial<SnowpackConfig['installOptions']>, EnvVarReplacements> {
  help?: boolean; // display help text
  version?: boolean; // display Snowpack version
  reload?: boolean;
  source?: SnowpackConfig['source'];
  config?: string; // manual path to config file
  env?: string[]; // env vars <--- overriding type here
}
```

Reply:
will deal with that tomorrow...

Reply:
Done